### PR TITLE
Improve: strategy-supertrend output acc. profit report to tsv file

### PR DESCRIPTION
**Pull request by andycheng123**

### CODE REVIEW

File: pkg/strategy/supertrend/strategy.go [hunk 1]
```diff
@@ -307,6 +310,18 @@ func (s *Strategy) PrintResult(o *os.File) {
...
 	hiyellow(f, "Last %d day(s) Accumulated Profit: %f\n", s.AccumulatedProfitLastPeriodWindow, s.dailyAccumulatedProfits.Sum())
 	hiyellow(f, "\n")
+
+	if s.AccumulatedProfitReportTsv != "" {
+		tf, err := os.OpenFile(s.AccumulatedProfitReportTsv, os.O_APPEND|os.O_WRONLY|os.O_CREATE, 0644)
+		if err != nil {
+			panic(err)
+		}
+		defer tf.Close()
+		csvwiter := csv.NewWriter(tf)
```
Comment (c9s) [inline pkg/strategy/supertrend/strategy.go:320]: use pkg/data/tsv  it supports tsv format as well

File: pkg/strategy/supertrend/strategy.go [hunk 2]
```diff
@@ -92,14 +200,7 @@ type Strategy struct {
...
 	// Accumulated profit report
-	accumulatedProfit   fixedpoint.Value
-	accumulatedProfitMA *indicator.SMA
-	// AccumulatedProfitMAWindow Accumulated profit SMA window
-	AccumulatedProfitMAWindow int `json:"accumulatedProfitMAWindow"`
-	dailyAccumulatedProfits   types.Float64Slice
-	lastDayAccumulatedProfit  fixedpoint.Value
-	// AccumulatedProfitLastPeriodWindow Last period window of accumulated profit
-	AccumulatedProfitLastPeriodWindow int `json:"accumulatedProfitLastPeriodWindow"`
+	AccumulatedProfitReport AccumulatedProfitReport `json:"accumulatedProfitReport"`
```
Comment (c9s) [inline pkg/strategy/supertrend/strategy.go:203]: use pointer type for this field? and check it in the Run method? e.g. `if x == nil { x = newXXX }`

File: pkg/strategy/supertrend/strategy.go [hunk 3]
```diff
@@ -30,6 +29,115 @@ func init() {
 	bbgo.RegisterStrategy(ID, &Strategy{})
 }
 
+// AccumulatedProfitReport For accumulated profit report output
+type AccumulatedProfitReport struct {
```
Comment (c9s) [inline pkg/strategy/supertrend/strategy.go:33]: looks like interval profit collector

File: pkg/strategy/supertrend/strategy.go (hunk 2 above)
Comment (andycheng123) [reply pkg/strategy/supertrend/strategy.go]: done

**Review by c9s — APPROVED** [review]

**Pull request merged**